pkg/R/DataFrame.R

```diff
@@ -768,6 +798,20 @@ setMethod("select", signature(x = "DataFrame", col = "Column"),
             dataFrame(sdf)
           })
 
+setMethod("select",
+          signature(x = "DataFrame", col = "list"),
```

Comment:
There isn't one in Scala or Python that matches. I created it as utility because it allows you to pass a list of arguments to `select`. Since the generic for `select` is currently specified as `function(x, col, ...)` where `col` is either `character` or `Column`, I don't see a good way to pass in an entire list.

In the Python/Scala API, you get around this by using `*args`, but that isn't quite as easy with S4.

You can see how this is used in the `withColumnRenamed` below.